src/globus_sdk/services/flows/client.py

```diff
@@ -270,6 +276,143 @@ def list_flows(
 
         return IterableFlowsResponse(self.get("/flows", query_params=query_params))
 
+    @_flowdoc("Update Flow", "Flows/paths/~1flows~1{flow_id}/put")
+    def update_flow(
+        self,
+        flow_id: UUIDLike,
+        title: str | None = None,
+        definition: dict[str, t.Any] | None = None,
+        input_schema: dict[str, t.Any] | None = None,
+        subtitle: str | None = None,
+        description: str | None = None,
+        flow_owner: str | None = None,
+        flow_viewers: list[str] | None = None,
+        flow_starters: list[str] | None = None,
+        flow_administrators: list[str] | None = None,
+        keywords: list[str] | None = None,
+        additional_fields: dict[str, t.Any] | None = None,
+    ) -> GlobusHTTPResponse:
+        """
+        Update a Flow
+
+        Only the parameter `flow_id` is required.
+        Any fields omitted from the request will be unchanged
+
+        :param flow_id: The ID of the Flow to fetch
+        :type flow_id: str or UUID
+        :param title: A non-unique, human-friendly name used for displaying the
+            flow to end users.
+        :type title: str (1 - 128 characters), optional
+        :param definition: JSON object specifying flows states and execution order. For
+            a more detailed explanation of the flow definition, see
+            `Authoring Flows <https://globus-automate-client.readthedocs.io/en/latest/authoring_flows.html>`_
+        :type definition: dict, optional
+        :param input_schema: A JSON Schema to which Flow Invocation input must conform
+        :type input_schema: dict, optional
+        :param subtitle: A concise summary of the flow’s purpose.
+        :type subtitle: str (0 - 128 characters), optional
+        :param description: A detailed description of the flow's purpose for end user
+            display.
+        :type description: str (0 - 4096 characters), optional
+        :param flow_owner: An Auth Identity URN to set as flow owner; this must match
+            the Identity URN of the entity calling `update_flow`
```

Comment:
"the" identity URN, or "an" identity URN of any of the caller's linked accounts?

Reply:
Hadn't considered this.
Lemme verify that it is in fact any identity associated with the new-owner account.

Good callout

Reply:
Did some testing, `"the Identity URN"` is correct.

Linked accounts are not allowed to set each other as the owner of a flow.

---

Given the following:
3 identities:
* A-1
* A-2
* B

where A-1 & A-2 are linked identities

I did the following:
1. As B: Create a flow (B is implicitly owner)
2. As B: Add A-1, A-2, B as Administrators of the flow
3. As A-1, attempt to set A-2 as FlowOwner -> Error 400
   * ```
      "Invalid value for field 'owner': if present it must match the identity of the calling user (<A-1's URN>) and that user must be an Administrator of the Flow"
      ```
4. As A-1, attempt to set A-1 as Flow Owner -> success
5. As A-1, attempt again to set A-2 as Flow Owner -> Same 400 as in step (3)